packages/amplify-category-api/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.js

```diff
@@ -372,8 +447,165 @@ async function askSecurityQuestions(context, parameters) {
   const advancedSettingsAnswer = await inquirer.prompt([advancedSettingsQuestion]);
 
   if (advancedSettingsAnswer.advancedSettings) {
+    authConfig = await askAdditionalAuthQuestions(context, parameters, authConfig, defaultAuthType);
+    if(process.env.SYNC === "true") {
```

Comment:
I think a more unique amplify specific name for the env variable would be better.